app/services/integrations/aggregator/invoices/base_service.rb

```diff
@@ -59,6 +59,8 @@ def item(fee)
             subscription_item
           end
 
+          return {} unless mapped_item
```

Comment:
In case it is nil (nothing mapped, not even the fallback item defined) we want to return an empty hash.

This code below will fail:

```
          {
            'item' => mapped_item.external_id,
            'account' => mapped_item.external_account_code,
            'quantity' => fee.units,
            'rate' => fee.precise_unit_amount
          }
```

```
mapped_item.external_id # => undefined method `external_id' for nil (NoMethodError)
```